### PR TITLE
Change bot replies to reflect new caps

### PR DESCRIPTION
**Pull request by Dracologist**
I decided to try to change the standard investment confirmation to include how many times you've invested in that post and the maximum number of times you can invest in the same post. I also tried to make the rejection message for investments above the maximum amount more informative. 

You might want to be pretty thorough in checking my code if you decide you want to use these because I have limited experience with Python

### CODE REVIEW

**Comment by Dracologist** [comment]
I decided to try my hand at creating commands to automatically invest the minimum or maximum amount of MemeCoins allowed because why not? If nothing else, it's decent practice for me to learn more about Python

**Comment by Dracologist** [comment]
I'm just adding small quality of life features me and people from my firm think might be cool because this is fun. Feel free to reject any changes you don't like

File: src/comment_worker.py [hunk 1]
```diff
@@ -170,7 +170,7 @@ def __call__(self, comment):
 
         # Parse the comment body for a command
         for reg in self.regexes:
-            matches = reg.fullmatch(comment.body.strip())
+            matches = reg.fullmatch(re.sub(r"![Ii]\s+", "!invest ", comment.body.strip()))
```
Comment (thecsw) [inline src/comment_worker.py]: Why are you manually setting this here?

File: src/message.py [hunk 2]
```diff
@@ -271,6 +274,17 @@ def modify_min_invest(minim):
...
+The maximum possible investment is %MAX% MemeCoins (50% of your balance) or 1,000,000,000 memecoins, whatever is higher.
+"""
+
+
+def modify_max_invest(maxim):
+    return MAX_INVEST_ORG.\
+        replace("%MAX%", format(int(maxim), ",d"))
+
+
+
```
Comment (thecsw) [inline src/message.py]: Too many empty lines!

**Review by thecsw — CHANGES_REQUESTED** [review]
I like the PR update and making the new feature more agile. So, here is the list of changes I think would be nice:
 - I am not sure we need to have max/min in the regex
 - There are some empty lines in `message.py`

**Review by thecsw — COMMENTED** [review]
If we set the limit to 50%, they could do just 50%

Money investing is not a casino where you can just go *max* or *ALL IN*. That is why we never made `!invest all` or `!all`, it just doesn't make sense in a economical situtation. 

If you have thoughts about this, I would love to chat

**Comment by Dracologist** [comment]
I'm pretty sure real-life investing doesn't have any limit to how much of your money you're allowed to invest in a single stock either, but adding that limit is probably going to improve r/MemeEconomy for most users. That's kinda my logic behind the `max` thing. It's a lot smaller, but it's convenient for when you feel like putting as much as you can into a meme.

I'll remove it from this branch and make a new branch off of this branch with the max and min

**Comment by hazzakak** [comment]
What else is wrong with this pr?

**Review by thecsw — APPROVED** [review]
Seems fine to me! This PR is approved. It will be merged if the mods or r/MemeEconomy decide to or when we start a new season with bonding stocks update. Thank you!

**Comment by thecsw** [comment]
@HarryjosephDevelopment Everything seems fine. Just approved it.

**Comment by hazzakak** [comment]
> @HarryjosephDevelopment Everything seems fine. Just approved it.

Sounds good! Is there a list of updates which will come out in the next season?

**Comment by thecsw** [comment]
@HarryjosephDevelopment the dream is to implement the stocks update from #373 and implement the codebase in Go instead of Python #394 

Also, @Keanu73 and I are talking about a possibility of rebuilding the bot with JS, see #394 for more details

**Comment by luisbg** [comment]
I like this a lot more than the original PR 😄

**Comment by luisbg** [comment]
> Also, @Keanu73 and I are talking about a possibility of rebuilding the bot with JS, see #394 for more details

What are the benefits of rebuilding in JS? I'm curious.

**Comment by thecsw** [comment]
Just to clarify, the changes are approved but they *WILL NOT* be merged until the necessity comes.

**Pull request closed without merging**